client/make-game.html

```diff
@@ -33,16 +33,23 @@ <h1>Create a new game</h1>
           All players will receive an invitation email with a personal
           link to the game.  The first player will start.
         </p>
+        <fieldset>
+          <legend>Size</legend>
+          <select name="size">
+            <option>Regular</option>
```

Comment:
please `selected="selected"` to make "Regular" the default.

Reply:
I think the first option is already the default selection (just like German is for language). Your suggestion does not look like valid HTML. Are you sure?

Reply:
done